src/python/tests/core/bot/fuzzers/libFuzzer/libfuzzer_launcher_integration_test.py

```diff
@@ -766,13 +768,15 @@ def setUp(self):
     environment.set_value('QUEUE_OVERRIDE', 'FUCHSIA')
     environment.set_value('OS_OVERRIDE', 'FUCHSIA')
     environment.set_value('FUCHSIA_BUILD_URL',
-                          'gs://fuchsia-clusterfuzz-test-july-15-2019/*')
+                          'gs://fuchsia-clusterfuzz-test-august-12-2019/*')
     self.tmp_resources_dir = tempfile.mkdtemp()
     environment.set_value('RESOURCES_DIR', self.tmp_resources_dir)
 
   def tearDown(self):
     shutil.rmtree(self.tmp_resources_dir, ignore_errors=True)
 
+  @skipIf(True,
```

Comment:
nit: use an env var here, e.g.

@skipIf(not environment.get_value('FUCHSIA_TESTS'))

Reply:
I fixed this.